rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.78.0"
+channel = "1.79.0"
```

Comment:
Is this bumped intentionally?

Reply:
Oops, that accidentally got in. I will revert.